packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -139,18 +139,18 @@ const useRemoteParticipantsWithCustomDisplayNames = (
   remoteParticipants: VideoGalleryRemoteParticipant[],
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback
 ): VideoGalleryRemoteParticipant[] => {
-  const newParticipants = remoteParticipants;
   const userIds = useMemo(() => {
-    return newParticipants.map((p) => p.userId);
-  }, [newParticipants]);
+    return remoteParticipants.map((p) => p.userId);
+  }, [remoteParticipants]);
 
   const avatarPersonaData = useCustomAvatarPersonaData(userIds, onFetchAvatarPersonaData);
-  newParticipants.forEach((p, i) => {
+  const newParticipantData = remoteParticipants.map((p, i) => {
```

Comment:
nitty nit: Avoid suffixes like `data`, `info` etc that don't add anything to the meaning beyond what the rest of the name already says.

I'd just name this `newParticipants`.